libraries/AP_RCProtocol/AP_RCProtocol.cpp

```diff
@@ -40,6 +40,7 @@ void AP_RCProtocol::init()
     backend[AP_RCProtocol::PPM] = new AP_RCProtocol_PPMSum(*this);
     backend[AP_RCProtocol::IBUS] = new AP_RCProtocol_IBUS(*this);
     backend[AP_RCProtocol::SBUS] = new AP_RCProtocol_SBUS(*this, true);
+    backend[AP_RCProtocol::SBUS_200kbaud] = new AP_RCProtocol_SBUS(*this, true, 200000);
```

Comment:
I have a feeling this may be a problem for the IOMCU RAM-wise

Reply:
Should I gate this behind `#if AP_RC_PROTOCOL_SBUS_200KBAUD_ENABLED`?

Reply:
I'm not sure - @tridge would know

Reply:
This now isn't built on the IOMCU.  tridge and I have discussed ways to reduce memory usage on the IOMCU and when that's done we can start to add these in.

Potentially we could wrap more of these options in `#ifdef`s so a user could easily customise a hwdef to just have the protocol they care about in there.